modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/operation/OperationMessageProcessorTestCase.java

```diff
@@ -437,8 +436,8 @@ public void precalculateExecutionContextForInterceptedProcessor() throws MuleExc
       context.set(spy((PrecalculatedExecutionContextAdapter) ctx));
     });
 
-    messageProcessor.process(quickCopy(event, ImmutableMap.of(INTERCEPTION_RESOLVED_CONTEXT, context.get(),
-                                                              "core:interceptionComponent", messageProcessor)));
+    messageProcessor.process(quickCopy(event, of(INTERCEPTION_RESOLVED_CONTEXT, context.get(),
```

Comment:
keep immutability

Reply:
same as before. Immutability will be still be provided by the quick event